Requested by upstream to package. Fixes a visual glitch that prevents
the new header appearance from working when using the Breeze Dark plasma
theme.

Upstream commit: https://invent.kde.org/frameworks/plasma-framework/-/commit/5be495b4d7bdeb6797d483980f3ca273e56b0a19

From 5be495b4d7bdeb6797d483980f3ca273e56b0a19 Mon Sep 17 00:00:00 2001
From: Chris Holland <zrenfire@gmail.com>
Date: Tue, 9 Jun 2020 20:20:12 -0400
Subject: [PATCH] Symlink widgets/plasmoidheading.svgz in breeze light/dark

PlasmoidHeading.qml only draws "widgets/plasmoidheading" if
fromCurrentTheme=true, so Breeze Light/Dark were not drawing it.
---
 src/desktoptheme/breeze-dark/CMakeLists.txt                | 5 ++++-
 src/desktoptheme/breeze-dark/widgets/plasmoidheading.svgz  | 1 +
 src/desktoptheme/breeze-light/CMakeLists.txt               | 5 ++++-
 src/desktoptheme/breeze-light/widgets/plasmoidheading.svgz | 1 +
 4 files changed, 10 insertions(+), 2 deletions(-)
 create mode 120000 src/desktoptheme/breeze-dark/widgets/plasmoidheading.svgz
 create mode 120000 src/desktoptheme/breeze-light/widgets/plasmoidheading.svgz

diff --git a/src/desktoptheme/breeze-dark/CMakeLists.txt b/src/desktoptheme/breeze-dark/CMakeLists.txt
index 7f8f99062..13a448a01 100644
--- a/src/desktoptheme/breeze-dark/CMakeLists.txt
+++ b/src/desktoptheme/breeze-dark/CMakeLists.txt
@@ -8,4 +8,7 @@ install(
     FILES colors "${CMAKE_CURRENT_BINARY_DIR}/metadata.desktop"
     DESTINATION ${PLASMA_DATA_INSTALL_DIR}/desktoptheme/breeze-dark
 )
-
+install(
+    DIRECTORY widgets
+    DESTINATION ${PLASMA_DATA_INSTALL_DIR}/desktoptheme/breeze-dark
+)
diff --git a/src/desktoptheme/breeze-dark/widgets/plasmoidheading.svgz b/src/desktoptheme/breeze-dark/widgets/plasmoidheading.svgz
new file mode 120000
index 000000000..b20bf0773
--- /dev/null
+++ b/src/desktoptheme/breeze-dark/widgets/plasmoidheading.svgz
@@ -0,0 +1 @@
+../../default/widgets/plasmoidheading.svgz
\ No newline at end of file
diff --git a/src/desktoptheme/breeze-light/CMakeLists.txt b/src/desktoptheme/breeze-light/CMakeLists.txt
index 5e595933f..9bb996a80 100644
--- a/src/desktoptheme/breeze-light/CMakeLists.txt
+++ b/src/desktoptheme/breeze-light/CMakeLists.txt
@@ -8,4 +8,7 @@ install(
     FILES colors "${CMAKE_CURRENT_BINARY_DIR}/metadata.desktop"
     DESTINATION ${PLASMA_DATA_INSTALL_DIR}/desktoptheme/breeze-light
 )
-
+install(
+    DIRECTORY widgets
+    DESTINATION ${PLASMA_DATA_INSTALL_DIR}/desktoptheme/breeze-light
+)
diff --git a/src/desktoptheme/breeze-light/widgets/plasmoidheading.svgz b/src/desktoptheme/breeze-light/widgets/plasmoidheading.svgz
new file mode 120000
index 000000000..b20bf0773
--- /dev/null
+++ b/src/desktoptheme/breeze-light/widgets/plasmoidheading.svgz
@@ -0,0 +1 @@
+../../default/widgets/plasmoidheading.svgz
\ No newline at end of file
-- 
2.27.0

